support/cas-server-support-oauth/src/main/java/org/apereo/cas/client/CasBackchannelConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package org.apereo.cas.client;
+
+import org.apereo.cas.CentralAuthenticationService;
+import org.jasig.cas.client.validation.TicketValidator;
+import org.pac4j.cas.config.CasConfiguration;
+import org.pac4j.core.context.WebContext;
+
+/**
+ * This is a configuration for pac4j CAS client that uses {@link CasBackchannelTicketValidator}.
+ *
+ * @author Kirill Gagarski
+ * @since 6.1.0
+ */
+public class CasBackchannelConfiguration extends CasConfiguration {
+
+    private final CentralAuthenticationService cas;
+
+    public CasBackchannelConfiguration(final String loginUrl, final CentralAuthenticationService cas) {
+        super(loginUrl);
+        this.cas = cas;
+    }
+
+    @Override
+    public TicketValidator retrieveTicketValidator(final WebContext context) {
+        return new CasBackchannelTicketValidator(cas);
```

Comment:
Wouldn't you want to set this directly on the CasConfirmation object without subclassing it? There is a defaultTicketValidator field of some kind where you could just set the validator on it, no?

Reply:
Yeah, you are probably right.